index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
+			<h1 class="projects_label">
+				Projects
+			</h1>
+			<div class="project_card">
+				<div class="project_image"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+			<div class="project_card">
+				<div class="project_image_1"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about_section">
+			<h1 class="about_title">
+				About
+			</h1>
+
+			<p class="about_paragraph">
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+				dolore magna
+				aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+			</p>
+
+			<div class="resume_button">
+				<p class="get_cv_text">
+					Get my CV
+				</p>
```

Comment:
- [ ] This should be a button. Please change it. And apply button styling. (active, hover, pressed).